Geometry/HGCalCommonData/data/TB181/June18/hgcalBeam.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../../../DetectorDescription/Schema/DDLSchema.xsd">
```

Comment:
@bsunanda @ianna @cvuosalo this change refers to the fact that there are additional subdirectories in the path defining the location of this file?

Reply:
Yes it does. Also I missed one for hgcProdcuts.xml which I have now corrected